src/masonite/tests/HttpTestResponse.py

```diff
@@ -147,9 +146,12 @@ def assertSessionHasErrors(self, keys=[], driver="cookie"):
         """Assert that session contains errors for the given list of keys (meaning that each given key
         exists in 'errors' dict in session.) If no keys are given this will assert that the
         sessions has errors without checking specific keys."""
-        errors = self.request.session.driver(driver).get("errors")
-        assert errors
+        session = self.request.session.driver(driver)
+        assert session.has("errors")
         if keys:
+            errors = session._get_serialization_value(
```

Comment:
Should we put this snippet as method of the session cookie driver to get a cookie unencrypted ? Because currently CookieDriver.get("cookie_name") returns the cookie encrypted if the EncryptCookie middleware has been used.

Reply:
Yeah thats fine